test/test-formatters.bash

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+find formatters/ -type f,l ! -name "*.formatted" | while read f;
```

Comment:
> `-type f,l`

Do we want to find symlinks, or just files?

Reply:
I've used symlinks for modes that are likely to be working on the same code with the same formatter e.g. rust-mode and rustic-mode both work on Rust and likely use rustfmt. I can strengthen the requirement if you like.

Reply:
Fair enough, that makes sense.